packages/scans/package.json

```diff
@@ -25,20 +27,31 @@
     "url": "https://github.com/zaproxy/actions-common/issues"
   },
   "dependencies": {
-    "@actions/artifact": "^0.2.0",
-    "@actions/core": "^1.1.1",
-    "@actions/exec": "^1.0.0",
-    "@actions/github": "^1.0.0",
+    "@actions/artifact": "^1.1.0",
+    "@actions/github": "^5.1.1",
     "adm-zip": "^0.4.14",
-    "d3-dsv": "^1.2.0",
-    "github-api": "^3.3.0",
-    "js-yaml": "^3.13.1",
     "lodash": "^4.17.15",
-    "s": "^1.0.0"
+    "request": "^2.88.2"
```

Comment:
This was meant to be removed, IMO it could be done in this PR.

Reply:
What exactly? The dependency is used. I would prefer to not add actual content changes to this PR and do them separately instead.

Reply:
I can't guarantee that I'll get to it - my current priority is setting up a test framework and formatting for the issue created by the action. The test setup should make it easier for others to do the change on this library, though.
What would be the consequence of not fixing the dependency? Would you remove the actions from the GitHub Actions marketplace?